crates/sui-transaction-builder/Makefile

```diff
@@ -0,0 +1,3 @@
+test-setup:
+	cd tests/test_example_v1 && sui move build --dump-bytecode-as-base64 >> ../../test_example_v1_build_output.json
+	cd tests/test_example_v2 && sui move build --dump-bytecode-as-base64 >> ../../test_example_v2_build_output.json 
```

Comment:
I don't see these files either:
- added to the repo (which if we're generating them we don't need)
- ignored via .gitignore so that they don't get added to the repo.

Reply:
When you refer to these files you're suggesting these two files `test_example_v1_build_output.json` + v2?
We don't need to check them in, they are generated on each run? 

I will add them to `.gitignore`.